GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/UI/VFSStatusBarItem.m

```diff
@@ -0,0 +1,100 @@
+#import "VFSAboutWindowController.h"
+#import "VFSStatusBarItem.h"
+#import "VFSProductInfoFetcher.h"
+
+@interface VFSStatusBarItem ()
+
+@property (strong, nonnull) NSStatusItem *statusItem;
+@property (strong, nonnull) VFSAboutWindowController *aboutWindowController;
+
+@end
+
+@implementation VFSStatusBarItem
+
+- (instancetype _Nullable)initWithAboutWindowController:(VFSAboutWindowController *_Nonnull)aboutWindowController
+{
+    if (self = [super init])
+    {
+        if (!aboutWindowController)
+        {
+            self = nil;
+        }
+        else
+        {
+            _aboutWindowController = aboutWindowController;
+        }
+    }
+    
+    return self;
+}
+
+- (void)load
+{
+    self.statusItem = [[NSStatusBar systemStatusBar]
+                       statusItemWithLength:NSVariableStatusItemLength];
+    
+    [self.statusItem setHighlightMode:YES];
+    
+    [self addStatusButton];
+    [self addMenuItems];
+}
+
+- (IBAction)handleMenuClick:(id)sender
+{
+    switch (((NSButton *) sender).tag)
+    {
+        case 0:
+        {
+            [self displayAboutBox];
+            break;
+        }
+        
+        default:
+        {
+            break;
+        }
+    }
+}
+
+- (IBAction)displayAboutBox
+{
+    [[NSApplication sharedApplication] activateIgnoringOtherApps:YES];
+    [self.aboutWindowController showWindow:self];
+    [self.aboutWindowController.window makeKeyAndOrderFront:self];
+}
+
+- (NSMenu *_Nullable)getStatusMenu
+{
+    return self.statusItem.menu;
+}
+
+- (void)addStatusButton
+{
+    NSImage *image = [NSImage imageNamed:@"StatusItem"];
+    
+    [image setTemplate:YES];
+    
+    [self.statusItem.button setImage:image];
+    [self.statusItem.button setTarget:nil];
+    [self.statusItem.button setAction:nil];
+}
+
+- (void)addMenuItems
+{
+    NSUInteger index = 0;
+    NSMenu *menu = [[NSMenu alloc] init];
+    NSMenuItem *aboutItem = [[NSMenuItem alloc]
+                             initWithTitle:@"About VFS For Git"
```

Comment:
Any plans for localization? All display strings should be in strings files for that.

Reply:
`VFSForGit` is English only.